app/controllers/api/v1/targets_controller.rb

```diff
@@ -0,0 +1,33 @@
+module Api
+  module V1
+    class TargetsController < Api::V1::ApiController
+      def index
+        @targets = Target.all
+        render json: @targets
+      end
+
+      def create
+        @target = current_user.targets.create!(target_params)
+        if @target.save
+          render json: { success: true, target: @target }
+        else
+          render json: { success: false }
+        end
```

Comment:
later, we could improve this approach.. 2 big things to do here (see together later): 
1- Add errors in case of failure.
2- `success` should came from the response status code